Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
```

Comment:
After this PR the main `wonnx` package is in the root whereas the python stuff is in a subdirectory. A more 'rusty' way of organising the repository is using a [workspace](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html). This basically means you have a directory for 'wonnx' (with its Cargo.toml), a directory for 'wonnx-python' (with its Cargo.toml) and a Cargo.toml in the root specifying 'wonnx' and 'wonnx-python' as workspace members.

Reply:
Yes, you're right. I'll fix that later.